files/build_templates/sonic_debian_extension.j2

```diff
@@ -654,6 +654,28 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker pull k8s.gcr
 echo "docker images pull complete"
 {% endif %}
 
+{% macro get_install_options(set_owner, enabled) -%}
```

Comment:
This section seems to be generic. I suggest slitting this PR into two:
1. appxt Infra work
2. porting dhcp_relay to appxt.